sdks/python/src/honcho/async_client/peer.py

```
@@ -132,7 +132,9 @@ async def chat(
             return None
         return response.content
 
-    async def get_sessions(self) -> AsyncPage[AsyncSession]:
+    async def get_sessions(
+        self, filter: dict[str, object] | None = None
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)